### PR TITLE
daemonset nodeuid init container: busybox wget => gnu wget

### LINKED ISSUE #92: Init container cannot connect to Kubernetes API server
Chart version: version: 1.1.14.
Kubernetes version 1.15(AWS EKS)

netdata-slave-* containers cannot start due to an error in `init-nodeuid` container.
```
>> kubectl logs netdata-slave-xxxxx init-nodeuid
wget: TLS error from peer (alert code 40): handshake failure
wget: error getting response: Connection reset by peer
```
It's related to this command:
```
wget -q -T 5 --no-check-certificate --header "${HEADER}" -O - "${URL}"
```
I've checked old busybox container(1.28) with the same result.

Mb, it'd better switch to `alpine` based image with `GNU wget`?

e.g.
```
...
        - name: init-nodeuid
          image: alpine:3.11
          volumeMounts:
            - name: nodeuid
              mountPath: "/nodeuid"
          env:
            - name: MY_NODE_NAME
              valueFrom:
                fieldRef:
                  fieldPath: spec.nodeName
          command:
            - "/bin/sh"
          args:
            - "-c"
            - '
            apk add --no-cache wget;
            TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token);
...
```

### PR DESCRIPTION
**Pull request by ilyam8**
fixes #92

use alpine-based image with installed `wget` instead of busybox

### CODE REVIEW

**Comment by ilyam8** [comment]
i tested it on gke, it works, @knatsakis please review

File: templates/daemonset.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ spec:
       {{- end }}
         {{- if .Values.slave.persistUniqueID }}
         - name: init-nodeuid
-          image: busybox
+          image: ilyam8/wget
```
Comment (knatsakis) [inline templates/daemonset.yaml:59]: Can this be `netdata/wget` please?

Comment (ilyam8) [reply templates/daemonset.yaml:59]: sure

**Comment by ilyam8** [comment]
i changed image to [netdata/wget](https://hub.docker.com/repository/docker/netdata/wget) @knatsakis

**Pull request merged**